src/js/components/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import TextField from '../TextField';
+import Button from '../Button';
+import styles from './LoginPage.less';
+
+const coursesLink = props => <Link to="/courses" {...props} />;
```

Comment:
in another component you create smth similar using a class method, please stick to one style